lib/internal/webstreams/readablestream.js

```diff
@@ -1020,7 +1020,9 @@ class ReadableStreamDefaultController {
   [kType] = 'ReadableStreamDefaultController';
 
   constructor() {
-    throw new ERR_ILLEGAL_CONSTRUCTOR();
```

Comment:
Maybe instead we could try lazy loading this?

Reply:
what do you mean? lazy loading what?

Reply:
Lazy loading the error construction, similar to here https://github.com/nodejs/node/blob/b68e5e798138be0041ba9ace72d8d45e63c068a1/lib/internal/webstreams/readablestream.js#L161

Reply:
what's the value of making it lazy? (as it will just add noise in my opinion and no stack trace from userland)

Reply:
hmmm i thought might be a viable soln. to get the tests fixed as well as get the perf benefit, in terms of stack trace not so sure i think this one is a obscure constructor no less likely people would call it. anyway glad you found a way but donno if these changes are a acceptable degree of deviation from the WHATWG spec

Reply:
if they were deviation the wpt tests would fail, no?

Reply:
presently they might not be strict enough, i remember in one of my prs adding an extra undefined somewhere were causing the tests to fail, again not the best person to comment anyway 😅

Reply:
talked about this offline